package.json

```diff
@@ -6,7 +6,8 @@
   "repository": "fusionjs/eslint-config-fusion",
   "main": "index.js",
   "dependencies": {
-    "eslint-config-uber-universal-stage-3": "^1.0.0-rc.7"
+    "eslint-config-uber-universal-stage-3": "^1.0.0-rc.7",
+    "eslint-plugin-import": "^2.8.0"
```

Comment:
Plugins need to be peer/dev deps so that eslint can resolve them from node_modules. Configs can be nested (if you use `require.resolve`).

https://github.com/eslint/eslint/issues/3458